docs/03_Software/07_FAIR_checklist/checklist.md

```diff
@@ -0,0 +1,94 @@
+# FAIR for research software guidelines
+
+To ensure your research software is findable and usable by other researchers, we have compiled a checklist to improve its FAIRness (Findable, Accessible, Interoperable, Reproducible).
+
+
+## Checklist
+
+The checklist below is in part based on the checklist provided by the [eScience Center](https://guide.esciencecenter.nl/#/nlesc_specific/checklist_matrix), licensed under CC BY 4.0.
+
+#### Version Control  
+_Essential_  
+- [ ] Use [git](https://www.atlassian.com/git) as a version control system 
+- [ ] Upload your project on [GitHub](https://github.com/) or [TU Delft GitLab](https://gitlab.tudelft.nl/)
+
+_Recommended_  
+- [ ] Make your repository [public](https://coderefinery.github.io/social-coding/social_coding/)
+- [ ] [Branch hygiene](https://coderefinery.github.io/git-branch-design/)
+- [ ] Use a branching model ([GitFlow](https://www.atlassian.com/git/tutorials/comparing-workflows/gitflow-workflow))
+- [ ] [Meaningful commit messages](https://www.git-scm.com/book/en/v2/Distributed-Git-Contributing-to-a-Project#_commit_guidelines)
+
+
+#### Documentation - Repository
+_Essential_  
+- [ ] [README](https://github.com/18F/open-source-guide/blob/18f-pages/pages/making-readmes-readable.md)
+- [ ] [LICENSE](https://doi.org/10.5281/zenodo.4629662)
+- [ ] [CITATION](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-citation-files)
+
+
+_Recommended_  
+- [ ] Make use of [Github issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues)
+- [ ] Contribution [guidelines](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/setting-guidelines-for-repository-contributors)
+- [ ] [Code of conduct](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-code-of-conduct-to-your-project)
+
+#### Documentation - Software
+_Essential_  
+- [ ] Source code documentation ([docstrings](https://numpydoc.readthedocs.io/en/latest/format.html))
+- [ ] Installation instructions
+- [ ] Usage documentation
+- [ ] Developer setup
+
+_Recommended_  
+- [ ] Examples and tutorials (Jupyter notebooks, videos, screencasts)
+- [ ] Website ([github.io pages](https://pages.github.com/), [Readthedocs](https://readthedocs.org/), [JupyterBook](https://jupyterbook.org/intro.html)
+- [ ] Build [API reference](https://developer.lsst.io/python/numpydoc.html) from docstrings
+
+
+#### Testing
+_Recommended_  
+- [ ] [Defensive programming](https://swcarpentry.github.io/python-novice-inflammation/10-defensive/index.html)
+- [ ] Integration or build test
+- [ ] User installation test
+- [ ] Unit tests (coverage >70%)
+- [ ] Continuous integration (GitHub Actions)
```

Comment:
Add the below URL for information on Continuous Integration from the Turing way guide.
https://the-turing-way.netlify.app/reproducible-research/ci/ci-options.html